scss/_base_forms.scss

```diff
@@ -0,0 +1,228 @@
+// Form element styles
+@mixin vf-b-forms {
+
+  label {
+    cursor: pointer;
+    display: block;
+    margin-bottom: .25rem;
+
+    @each $state, $color in $states {
+      &.has-#{$state} {
+        color: $color;
+      }
+    }
+  }
+
+  // Input styles
+  input {
+
+    &[type='text'],
+    &[type='date'],
+    &[type='datetime'],
+    &[type='datatime-local'],
+    &[type='month'],
+    &[type='time'],
+    &[type='week'],
+    &[type='color'],
+    &[type='number'],
+    &[type='search'],
+    &[type='password'],
+    &[type='email'],
+    &[type='url'],
+    &[type='tel'] {
+      @extend %input-elements;
+    }
+
+    &[type='reset'] {
+      display: none;
+    }
+
+    &[type='search'] {
+      -webkit-appearance: none;
+      border-radius: 0;
+    }
+
+    // Checkbox and radio inputs
+    &[type='checkbox'],
+    &[type='radio'] {
+      @extend %tick-elements;
+
+      + label {
+        display: inline;
+        margin-left: .3125rem;
+        vertical-align: middle;
+        width: auto;
+      }
+    }
+
+    &[type='submit'] {
+      border: 0;
+      padding: .75rem 1.5rem;
+      background-color: $color-positive;
+      color: $color-light;
+
+      &:hover {
+        cursor: pointer;
+        background-color: darken($color-positive, 20%);
+      }
+    }
+  }
+
+  // Select styles
+  select {
+    @extend %input-elements;
+    background: $color-light url('#{$assets-path}04d2075a-chevron-down.svg') no-repeat;
```

Comment:
Icons should be included using Base64 as Vanilla should be able to be used offline.